test/dlc_tests/benchmark/ec2/pytorch/training/test_performance_pytorch_training.py

```diff
@@ -37,15 +43,41 @@ def execute_pytorch_gpu_py3_imagenet_ec2_training_performance_test(connection, e
 
     # Make sure we are logged into ECR so we can pull the image
     connection.run(f"$(aws ecr get-login --no-include-email --region {region})", hide=True)
-
     connection.run(f"nvidia-docker pull -q {ecr_uri}")
-
+    timestamp = time.strftime('%Y-%m-%d-%H-%M-%S')
+    log_name = f"imagenet_{os.getenv('CODEBUILD_RESOLVED_SOURCE_VERSION')}_{timestamp}.txt"
```

Comment:
Doesn't using this as the log name cause conflicts between multiple tests running in parallel, and make them overwrite the data the other produces in this file?